src/HybridApp/App.xaml.cs

```diff
@@ -5,7 +5,10 @@ public partial class App : Application
 	public App()
 	{
 		InitializeComponent();
-
-		MainPage = new MainPage();
 	}
+
+    protected override Window CreateWindow(IActivationState? activationState)
+    {
+        return new Window(new MainPage());
```

Comment:
Let's make it even snazzier and have a title:

```suggestion
        return new Window(new MainPage()) { Title = "eShop" };
```

Apparently this has been a bug in the .NET MAUI Blazor Hybrid templates for only... 3 years... and we just fixed it 😁